homeassistant/auth/auth_store.py

```diff
@@ -282,6 +281,19 @@ def async_log_refresh_token_usage(
             )
         self._async_schedule_save()
 
+    @callback
+    def async_edit_expiry_date(
+        self, refresh_token: models.RefreshToken, *, disable_expiry_date: bool
+    ) -> None:
+        """Edit epiry date from refresh token."""
+        if disable_expiry_date:
+            refresh_token.expire_at = None
+        else:
+            refresh_token.expire_at = (
+                refresh_token.last_used_at or dt_util.utcnow()
+            ).timestamp() + REFRESH_TOKEN_EXPIRATION
```

Comment:
This means we bump the expiry time every time this method is called, is that intentional or should it only happen if expire_at is None?